utils/apiAppleHelper.js

```diff
@@ -0,0 +1,25 @@
+/*
+Helper file for the Apple Endpoints
+*/
+
+const logger = require('./logger');
+
+exports.appleData = async (redis, keys) => await fetchAppleData(redis, keys);
+
+const fetchAppleData = async (redis, keys) => {
```

Comment:
can we just do 

```js
module.exports = async (redis, keys) => { ... }
```
also, why don't we just pass the key that is needed for the function (pass `keys.apple_all` instead of `keys`)